Makefile

```diff
@@ -1,6 +1,6 @@
 SHELL = /usr/bin/env bash -o pipefail
 
-REPO              ?= conduitio/conduit-operator
+REPO              ?= ghcr.io/conduitio/conduit-operator
```

Comment:
I'm wondering if it would be enough to replace this with `conduit.docker.scarf.sh/conduitio/conduit`.

Reply:
Sorry, bad place.:D Yes, that.

Reply:
PRs welcome ;-)

Reply:
Will do, was just double-checking with you.:)